modules/ROOT/assets/attachments/bundled-admin.yaml

```
@@ -4301,6 +4555,167 @@ paths:
         schema: *ref_3
         example: db1
         description: The name of the database to run the operation against.
+  /{db}/_config/audit:
+    get:
+      summary: Get database audit configuration
+      description: |-
+        Retrieve the audit configuration for the database specified.
+
+        Required Sync Gateway RBAC roles:
+
+        * TODO
```

It's fair enough having a TODO in the file but comment it out so no risk of it being exposed to users.

```suggestion
      # TODO: Required Sync Gateway RBAC roles
```